communication/best-practices.md

```diff
@@ -0,0 +1,82 @@
+# Mailing list and Github usage best practice                                                       
```

Comment:
We should encourage people to use https://gubernator.k8s.io/pr (and [improve it](https://github.com/kubernetes/test-infra/tree/master/gubernator)) rather than everyone (individually no less!) to setup some baroque set of gmail filters.

Reply:
I think that should be in another page, like having a detailed readme file about gubernator. 
Maybe, we could add a note about gubernator and that we recommend using it?

Reply:
+1 about adding a note about gubernator

Reply:
Any chance we can address this?

Reply:
Oups, I thought I added it the note already, sorry - updating it right now.